03-workflows-config.Rmd

```diff
@@ -22,38 +23,38 @@ These runtime variables are both the defaults for our Fred Hutch configuration a
 
 
 ## Fred Hutch Custom Runtime Variables
-For the `gizmo` cluster, the following runtime variables are available that are customized to our configuration.  
-What is specified below is the current default as written, you can edit these in the config file if you'd like OR you can specify these variables in your `runtime` block in each task to change only the variables you want to change from the default for that particular task.  
+For the `gizmo` cluster, the following custom runtime variables are available.  
+Below we show the current defaults. You can change these either by editing the config file, OR by specifying them in the `runtime` block for individual tasks in a WDL file.  Task-level specifications will override the defaults.  
```

Comment:
I'm not 100% sure we want people to edit the config file. Best to just advise to overwrite them in the runtime block only.  Otherwise SciComp can't help if someone has messed up their config. ;)

Reply:
sounds good to me.  That would resolve a couple of the confusions I mentioned [here](https://github.com/fhdsl/FH_WDL101_Cromwell/issues/24).   I'll remove mention of the config file here.

I guess I also ask, for this spot:   can you also change runtime variables for an entire workflow in a wdl file?   Or can you only do it per task?

Reply:
Yes, you CAN but not in WDL 101. ;) It's always ultimately at the task level, but there are ways you can write your workflow when you DO want to change it at the workflow level.  Probably out of scope for this class but better for WDL 102.